### ISSUE TITLE
Ability to run all tests in parallel

### ISSUE THREAD

**Issue by binarylogic** [timberio/vector-test-harness-github-actions-test-repo#6]
It looks like Github Actions has the ability to [run jobs as a matrix](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategy). I would like to run _every_ test case in parallel unless otherwise specified. For example:

* PR comment `/test` runs all tests in parallel.
* PR comment `/test -t docker_partial_events_merging_correctness` run a single test.
* PR comment `/test -t docker_partial_events_merging_correctness -t disk_buffer_performance` runs only the tests specified.

All tests are designed to be namespaced, there should be no problem running them all in parallel. I am also not concerned about resource usage or cost. If we bump into AWS limits we can request an increase.

**Comment by binarylogic** [comment]
I added a fourth requirement, which will allow us to run specific tests only. This, if I had to guess, will be how we run tests most of the time.

**Comment by MOZGIII** [comment]
So, what the `/test` command do? Should it just run tests and post a link to the results? Or should it also run `bin/compare` and plot the results?

**Comment by binarylogic** [comment]
Yep, have it run `bin/compare` for now. I plan to work on the output. The way you have it should make it very easy for me to customize.
